test/util/util.go

```diff
@@ -189,6 +191,55 @@ func wrapErrorWithCertificateStatusCondition(client clientset.CertificateInterfa
 	return pollErr
 }
 
+// WaitCertificateIssuedValid waits for the given Certificate to be
+// 'Ready' and ensures the stored certificate is valid for the specified
+// domains.
+func WaitCertificateIssuedValid(certClient clientset.CertificateInterface, secretClient corecs.SecretInterface, name string, timeout time.Duration) error {
```

Comment:
I see this new function added, but the old one (which is a method on the Framework) seems to still exist? (can't see it removed in this PR)